src/IO/DataReader.cs

```diff
@@ -226,19 +227,24 @@ internal string ReadASCII(int size)
         {
             EnsureSize(size);
 
-            StringBuilder sb = new StringBuilder(size);
-
-            for (int i = 0; i < size; i++)
+            ValueStringBuilder sb = new ValueStringBuilder(size);
             {
```

Comment:
Is this a naked context? I think this is generally not recommended unless you have a specific need for it since this can create another stack. In this particular case it doesn't look needed.

Reply:
done! ty